Doc/library/random.rst

```diff
@@ -111,6 +111,9 @@ Bookkeeping functions
    as an optional part of the API. When available, :meth:`getrandbits` enables
    :meth:`randrange` to handle arbitrarily large ranges.
 
+   .. versionchanged:: 3.9
+      This method now accepts ``0`` for *k*.
```

Comment:
```suggestion
      This method now accepts zero for *k*.
```

Reply:
Hmm... why not. Is there a consensus on this? I do not want to come back and revert to `0` again :-)

Reply:
Either way looks fine to me. I agree that the second reads _slightly_ more naturally, but only slightly.